docs/book/src/clusterctl/developers.md

```diff
@@ -138,6 +144,12 @@ EOF
 
 - Run `kind create cluster --config kind-cluster-with-extramounts.yaml` to create the management cluster using the above file
 
+  if you are using kind v0.8.0+
```

Comment:
What about reordering things so setting the env variable is the step before create cluster?

- Run the following command to create a kind config file for allowing the Docker provider to access Docker on the host:
```bash
...
```
- If you are using kind v0.8.0+, make sure that the management cluster and the workload cluster are created on the same network
```bash
  export KIND_EXPERIMENTAL_DOCKER_NETWORK=bridge
```
- Run `kind create cluster --config kind-cluster-with-extramounts.yaml` to create the management cluster using the above file

Reply:
Might be worthwhile to add tabs here and split between v0.7.x (first/default) and v0.8.x

Reply:
Could we also add a simple reason as to why this change is needed? That is, what does setting that variable do.

Reply:
added both tabs and simple reason